docs/docs/behaviors/mod-morph.md

```diff
@@ -71,3 +65,30 @@ Example:
 ```
 mods = <(MOD_LGUI|MOD_LSFT|MOD_RGUI|MOD_RSFT)>;
 ```
+
+### Advanced configuration
+
+`masked_mods`
+
+When a modifier specified in `mods` is being held, it won't be sent along with the morphed keycode if it is also part of `masked_mods`. To sent all modifiers along with the morphed keycode, set `masked_mods` to `<0>`. By default, `masked_mods` equals `mods`.
+
+For example, the following configuration morphs `LEFT_SHIFT` + `BACKSPACE` into `DELETE`, and morphs `RIGHT_SHIFT` + `BACKSPACE` into `RIGHT_SHIFT` + `DELETE`.
+
+```
+/ {
+    behaviors {
+        bspc_del: backspace_delete {
+            compatible = "zmk,behavior-mod-morph";
+            label = "BACKSPACE_DELETE";
+            #binding-cells = <0>;
+            bindings = <&kp BACKSPACE>, <&kp DELETE>;
+            mods = <(MOD_LSFT|MOD_RSFT)>;
+            masked_mods = <(MOD_LSFT)>;
+        };
+    };
+};
+```
+
+### Limitations
+
+In some circumstance, when the morphed keycode is a hold-tap, the modifier gets sent along with the hold-tap regardless of the specification of `masked_mods`.
```

Comment:
```suggestion
Under certain circumstances where the morphed keycode is a hold-tap, the modifier gets sent along with the hold-tap regardless of the `masked_mods` value.
```

Reply:
Actually, that part is hopefully obsolete given b903e4f3710d0f331407c6ad26ce466a3a310845. Still waiting for @nickconway to confirm before removing the note.

Reply:
It seems the fix worked and mod-morphing into hold-tap no longer ignores masked-mods. So I am dropping the disclaimer